cmd/get.go

```diff
@@ -0,0 +1,129 @@
+/*
+Copyright © 2022 NAME HERE <EMAIL ADDRESS>
+*/
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"npelection/app"
+	"os"
+	"strings"
+
+	"github.com/PuerkitoBio/goquery"
+	"github.com/olekukonko/tablewriter"
+	"github.com/spf13/cobra"
+)
+
+var (
+	district                  string
+	numParties, numCandidates int
+)
+
+// getCmd represents the get command
+var getCmd = &cobra.Command{
+	Use:   "get",
+	Short: "Get Election Data for certain district",
+	RunE: func(cmd *cobra.Command, _ []string) error {
+		// Get the link for the district to scrape
+		link, ok := districtLinkMap[district]
+		// If the district is not found, return an error
+		if !ok {
+			closestDistrict := app.GetClosestDistrict(district, districtLinkMap)
+			fmt.Printf("Incorrect District Name. Did you mean %s?\n", closestDistrict)
+			fmt.Printf("Use `npelection list` to get a list of all districts.\n\n")
+			cmd.Help()
+			return nil
+		}
+		// Get the HTML document
+		response, err := http.Get(link)
+		if err != nil {
+			e := fmt.Errorf("error getting response: %v", err)
+			return e
+		}
+		defer response.Body.Close()
+
+		if response.StatusCode != 200 {
+			e := fmt.Errorf("status code error- Got: %v Wanted: %v", response.StatusCode, http.StatusOK)
+			return e
+		}
+		// Load the HTML document
+		doc, err := goquery.NewDocumentFromReader(response.Body)
+		if err != nil {
+			e := fmt.Errorf("error parsing response body: %v", err)
+			return e
+		}
+
+		// Find the table with constituency wise data
+		constituencyTableHTML := doc.Find("body > section > section:nth-child(5) > div > div > div.gy-4 > div.election-2079 > div > div > div")
```

Comment:
For future improvement,
- Research if the website uses `election-{year}` for the past election data.
- If yes, maybe replace dynamically by getting current year so that the tool doesn't break just because of this mismatch class.

Reply:
No they have archived the previous election in a separate subdomain from what it previously was and it is in a different manner entirely.

Reply:
✅  Okay